program/print-numbers-from-1-to-n/PrintNumbersFrom1ToN.java

```diff
@@ -0,0 +1,12 @@
+public class PrintNumbersFrom1ToN{
+    public static void main(String[] args) {
+       Scanner sc = new Scanner(System.in);
+       System.out.print("Enter the number: ");
+       int n = sc.nextInt();
+       print(n);
+  }
+  static void print(int n){
+    for(int i = 1; i <=n; i++){
+      System.out.print(i + " ");
+  }
+}
```

Comment:
class closing bracket is missing .
Can you do an unit testing before submitting the PR for a review

Reply:
I really apologise for the inconvenience caused. I won't let it happen next time :)

Reply:
> I really apologise for the inconvenience caused. I won't let it happen next time :)

No need to apologize, we're here to help you.